### DISCUSSION TITLE
How to import 'got' node module to be used in background.js?

### DISCUSSION THREAD

**Discussion by realtebo** [quasarframework/quasar#11973] [CLI - BEX mode] [answered]
I cannot use "import" in background.js

I also cannot use "require" in this script.

So..... how can I import an external node module to be used in the background.js?

**Comment by MilosPaunovic** [comment, marked as answer]
[https://stackoverflow.com/questions/12575965/is-it-possible-to-develop-google-chrome-extensions-using-node-js](https://stackoverflow.com/questions/12575965/is-it-possible-to-develop-google-chrome-extensions-using-node-js)
[https://stackoverflow.com/questions/48175324/is-it-possible-to-use-npm-packages-in-chrome-extensions](https://stackoverflow.com/questions/48175324/is-it-possible-to-use-npm-packages-in-chrome-extensions)

**Reply by realtebo** [reply]
The second So question is telling me that it's possibile using webpack and quasar is using webpack. 

So, in this specific context, how can we use 'got' in background.js?
